synapse/module_api/__init__.py

```diff
@@ -1226,6 +1228,43 @@ async def sleep(self, seconds: float) -> None:
 
         await self._clock.sleep(seconds)
 
+    async def send_http_push_notification(
+        self,
+        user_id: str,
+        device_id: Optional[str],
+        content: JsonDict,
+        tweaks: Optional[JsonMapping] = None,
+    ) -> bool:
```

Comment:
This seems OK, but I question whether it is reasonable to expose this. It is insanely low-level for a module API.

Reply:
I've put a rational and some examples in the top comment.